ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -157,11 +157,20 @@ do
   yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" "$f"
 done
 
+if [ "${FIPS_ENABLED:-false}" = "true" ]; then
+	echo "Adding 'fips: true' to install-config.yaml"
+	yq --yaml-output --in-place ".
+		| .fips = true
+	" "$SHARED_DIR/install-config.yaml"
+fi
```

Comment:
This piece of work is already included in the `ipi-conf` step, so it can be removed from here.

Reply:
good to know. thanks